analyze-question-sentiment.go

```diff
@@ -57,29 +58,40 @@ type tagAnalysisResult struct {
 	negative          int
 	closed            int
 	closedAndNegative int
+
+	// min and max dates of actual items
+	minDate sql.NullTime
+	maxDate sql.NullTime
 }
 
-func mustParseTime(date string) time.Time {
+func mustParseDate(date string) time.Time {
```

Comment:
Is this even used any more?

Reply:
No. Would you generally prefer all unused things removed?

Reply:
Yes (there's always git history if needed). I'll just do it in a followup cleanup, so we can merge this.